src/commands/init/generateTemplate/index.js

```diff
@@ -73,7 +73,7 @@ function filterFiles(filters) {
 function render(template, data, callback) {
     let rendered;
     try {
-        rendered = Handlebars.compile(template)(data);
+        rendered = Handlebars.compile(template, data)(data);
```

Comment:
Do we want `{ noEscape: true }` for all the usages of `render`? In `renderTemplateFiles` it was previously used without it, as metadata was taken directly from metalsmith.
Maybe we should instead open up `render` to use separate `data` and `options` and pass `noEscape` in `options` only when rendering log message?